### PR TITLE
Topic recognition

### PR DESCRIPTION
**Pull request by ecutm1**
Improved UNet segmentation code to be merged into the recognition folder

thanks!
Elliott Cutmore
s4354061

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Sufficiently detailed commit messages
- Code valid for Normal Difficulty
- Results are good

Other Feedback
- Really good overall. (OPTIONAL) I'd break up some overly long function into small reusable chunks in the future. e.g. `def conv(params)` which can be reused over and over to build the model.

**Comment by rexsw** [comment]
Code looks good for normal difficulty

For next time;

Make sure you remove unused imports

File: recognition/Segment_UNet_s4354061_Elliott_Cutmore/driver_load_evaluate.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
+from segment import *
```
Comment (shakes76) [inline recognition/Segment_UNet_s4354061_Elliott_Cutmore/driver_load_evaluate.py:1]: Code header block should be added for all scripts/source files

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.

**Pull request merged**